scripts/pem_to_pub_c.py

```diff
@@ -42,29 +38,25 @@ def main():
             'Unusually large public exponent detected. ' +
             'OP-TEE handles only public exponents up to 2^32 - 1.')
 
-    f = open(args.out, 'w')
-
-    f.write("#include <stdint.h>\n")
-    f.write("#include <stddef.h>\n\n")
-
-    f.write("const uint32_t " + args.prefix + "_exponent = " +
-            str(key.publickey().e) + ";\n\n")
-
-    f.write("const uint8_t " + args.prefix + "_modulus[] = {\n")
-    i = 0
-    for x in array.array("B", long_to_bytes(key.publickey().n)):
-        f.write("0x" + '{0:02x}'.format(x) + ",")
-        i = i + 1
-        if i % 8 == 0:
-            f.write("\n")
-        else:
-            f.write(" ")
-    f.write("};\n")
-
-    f.write("const size_t " + args.prefix + "_modulus_size = sizeof(" +
-            args.prefix + "_modulus);\n")
-
-    f.close()
+    with open(args.out, 'w') as f:
+        f.write("#include <stdint.h>\n")
+        f.write("#include <stddef.h>\n\n")
+        f.write("const uint32_t " + args.prefix + "_exponent = " +
+                str(key.publickey().e) + ";\n\n")
+        f.write("const uint8_t " + args.prefix + "_modulus[] = {\n")
```

Comment:
this `f.write()` should be on top of next block.
or remove all empty lines (line 47 and 57).